lib/storage/index.native.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * On native we can leave almost everything to AsyncStorage, but files need to be prepared.
+ * Before files are saved in our key/value storage they should first be copied to
+ * our "local space" (App Documents) so that they're not lost - image pickers and document
+ * pickers would move files to cache which can be cleared before we've uploaded them
+ *
+ * We don't actually save the file to our key/value store but only the reference to it
+ */
+
+import Storage from './providers/AsyncStorage';
+import FileHandler from './providers/NativeFileHandler';
+
+const nativeStorage = {
+    ...Storage,
+
+    /**
+     * On native it's a no-op because it's not possible to have multiple instances of Onyx (e.g. multiple tabs)
+     */
+    synchronizeInstances() {},
```

Comment:
I am suggesting we put it there so that there is some default. We are overwriting it anyway and then we won't have to worry about "implementing" it here.

Reply:
I've added this change and removed the empty `synchronizeInstances` 

##### Onyx.init
```js
    if (shouldSyncMultipleInstances && _.isFunction(Storage.keepInstancesSync)) {
        Storage.keepInstancesSync(keyChanged);
    }
```